node/server/server.ts

```diff
@@ -64,17 +67,22 @@ const prepareServer = ({
       formatImportError,
     })
 
+    const features: Record<string, boolean> = {}
     const importMap = baseImportMap.clone()
-    const vendor = await vendorNPMSpecifiers({
-      basePath,
-      directory: distDirectory,
-      functions: functions.map(({ path }) => path),
-      importMap,
-      logger,
-    })
 
-    if (vendor) {
-      importMap.add(vendor.importMap)
+    if (featureFlags?.edge_functions_npm_modules) {
```

Comment:
I'm assuming that you're planning on statically setting this flag from CLI, or do you want to roll this out via DevCycle?

Reply:
Using DevCycle.